README.md

```diff
@@ -81,9 +81,16 @@ issues or [on Discord][discord].
 ## Maintainers
 
 Open Source Pledge is not involved in any flow of funds and so we do not
-directly onboard maintainers. We encourage you to onboard to the following
-platforms to receive funding from Open Source Pledge Members:
+directly onboard maintainers. Please confirm with each open source project
```

Comment:
Right — I've removed that sentence so that this section exclusively addresses maintainers. Presumably, sponsors will have to choose only out of those platforms that maintainers are present on anyway.